cryosparc2/protocols/protocol_cryorefine.py

```diff
@@ -67,7 +67,7 @@ def _defineParams(self, form):
                       label="Input particles", important=True,
                       validators=[Positive],
                       help='Select the input images from the project.')
-        form.addParam('referenceVolume', PointerParam, pointerClass='Volume',
+        form.addParam('refVolume', PointerParam, pointerClass='Volume',
```

Comment:
Renaming a params will break existing data compatibility. So, existing workflows (json) or saved data will not work.

Reply:
I agree, unless a very good reason I think it is better to keep existing param names in general